### PR TITLE
[MLv2] Filters — Add unit tests for `TimeFilterPicker`

### PR DESCRIPTION
**Pull request by kulyk**

### CODE REVIEW

File: frontend/src/metabase/common/components/FilterPicker/TimeFilterPicker/TimeFilterPicker.unit.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,340 @@
+import moment from "moment";
```
Comment (iethree) [inline frontend/src/metabase/common/components/FilterPicker/TimeFilterPicker/TimeFilterPicker.unit.spec.tsx:1]: will dayjs work here?

File: frontend/src/metabase/common/components/FilterPicker/TimeFilterPicker/TimeFilterPicker.unit.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+      onChange={onChange}
+      onBack={onBack}
+    />,
+  );
+
+  function getNextFilterParts() {
+    const lastCall = onChange.mock.calls[onChange.mock.calls.length - 1];
+    const [filter] = lastCall;
+    return Lib.timeFilterParts(query, 0, filter);
+  }
```
Comment (iethree) [inline frontend/src/metabase/common/components/FilterPicker/TimeFilterPicker/TimeFilterPicker.unit.spec.tsx:110]: this is lovely ❤️

**Review by iethree — APPROVED** [review]
This is great. Can we also test a few more cases:

- invalid user input like 26:62
- invalid filter value like greater than 26:62
- swapping the column when there's an existing filter

**Comment by kulyk** [comment]
Thanks for the excellent suggestions, @iethree ❤️ I think I've addressed everything, please take another look

**Review by iethree — APPROVED** [review]
💖

**Pull request merged**